lib/Controller.js

```diff
@@ -15,8 +16,10 @@ var Q = require('q');
 var qio = require('q-io/fs');
 var path = require('path');
 var util = require('util');
-var adapter = require('./Adapters/AdapterFactory.js').getAdapter();
+
 var MonitorServer=require('./MonitorServer.js');
+const events = require('events');
+
```

Comment:
nothing in this file really change, so for each merge revert all space changes so this file will not be in the pull reuqest